### ISSUE TITLE
Which link to post for my package

### ISSUE THREAD

**Issue by MaryHe** [STAT545-UBC/Discussion#256]
Hi there,

I pushed my package to a public repo. Just want to make sure is it this main link to  this repo (https://github.com/MaryHe/First-Package-of-Mary) that I need to submit?

Thanks!

Mary

**Comment by jennybc** [comment]
I'm going to quote from the [homework assignment](http://stat545-ubc.github.io/hw10_package.html):

> Push your package to GitHub, preferably a public repo.... Write your reflections in a document in your normal STAT 545 course repo and open the usual issue there. Make sure to provide a link to your package, so TAs/prof/peers can find it for review!

So YES in your "reflections" document and in your issue, please provide a clear link to the repo where we can find you package. So YES the one you provide above.

**Comment by MaryHe** [comment]
Hi Jenny,

Thanks so much for your quick reply!

Yet I got another question. I built my package from the very beginning again just to familiarise myself with the process. Yet I did not follow the instructions about connecting my local repo to github using `use_github` and the personal access token thing (I forgot this step). 

Instead, I created a package locally, and then I created a public repo on github, and then I followed the instruction on github, that is, to type this into my project shell
```
git remote add origin https://github.com/MaryHe/First-Package-of-Mary.git
git push -u origin master
```
And I successfully connected them. Hmmm.. Is this ok? Will this cause any problems for people to download my package? I am really confused about this type of technical things...

Thanks again!

Mary
